resources/views/game-details.blade.php

```diff
@@ -0,0 +1,28 @@
+
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Game Details</title>
+</head>
+<body>
+    <h1>Game Details</h1>
+    <table>
+        <thead>
+            <th>Details</th>
+        </thead>
+        <tbody>
+        @foreach($games as $game)
```

Comment:
why are you adding a game-details view that loops over a game list, just like the game-list view already does? we already have a game-info template.

Reply:
Is this whole view redundant now? Because the game-list accomplishes what it is asking for?

Reply:
yes we now have game-list (which we've always had) that shows the list of games, and game-info which shows a single game in a more-detailed view. i don't think we need game-details.